src/graphql/types/Organization/Organization.ts

```
@@ -46,5 +63,25 @@
+			description: "Custom fields associated with the organization",
+			nullable: false,
+			resolve: async (organization, _args, ctx) => {
+				interface CustomField {
+					id: string;
+					name: string;
+					type: string;
+					organizationId: string;
+				}
+				const customFields: CustomField[] =
+					await ctx.drizzleClient.query.customFieldsTable.findMany({
+						where: (fields: any, operators: any) =>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)